tests-frontend/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "tests-frontend",
+  "version": "1.0.0",
+  "description": "Frontent tests for Malwarecage",
+  "main": "cypress/integration/login.js",
+  "dependencies": {},
+  "devDependencies": {
+    "cypress": "^4.9.0",
+    "wait-on": "^5.0.1"
+  },
+  "scripts": {
+    "test": "wait-on http://malwarefront:3000 && cypress run"
+  },
+  "author": "dsk",
+  "license": "ISC"
```

Prevent `npm publish`, remove license and author note
```suggestion
  "private": true
```